Error in wizarr update script

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

wizarr

### 📂 What was the exact command used to execute the script?

update (within the LCX of wizarr)

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Wanted to update wizarr because of new release.
Release was grabbed but script failed in the end, see error output.
Update of the app is still successful tho.

### 🔄 Steps to reproduce the issue.

Using the update function within the LXC

### ❌ Paste the full error output (if available).

```bash
  ✔️   Stopped Wizarr
  ✔️   Backup Created
  ✔️   Setup uv 0.7.15
  ✔️   Deployed: wizarr (2025.6.5)
  ⏳   Updating Wizarr to v2025.6.5compiling catalog app/translations/it/LC_MESSAGES/messages.po to app/translations/it/LC_MESSAGES/messages.mo
compiling catalog app/translations/da/LC_MESSAGES/messages.po to app/translations/da/LC_MESSAGES/messages.mo
compiling catalog app/translations/fr/LC_MESSAGES/messages.po to app/translations/fr/LC_MESSAGES/messages.mo
compiling catalog app/translations/he/LC_MESSAGES/messages.po to app/translations/he/LC_MESSAGES/messages.mo
compiling catalog app/translations/pl/LC_MESSAGES/messages.po to app/translations/pl/LC_MESSAGES/messages.mo
compiling catalog app/translations/es/LC_MESSAGES/messages.po to app/translations/es/LC_MESSAGES/messages.mo
compiling catalog app/translations/hr/LC_MESSAGES/messages.po to app/translations/hr/LC_MESSAGES/messages.mo
compiling catalog app/translations/cs/LC_MESSAGES/messages.po to app/translations/cs/LC_MESSAGES/messages.mo
compiling catalog app/translations/pt_BR/LC_MESSAGES/messages.po to app/translations/pt_BR/LC_MESSAGES/messages.mo
compiling catalog app/translations/sv/LC_MESSAGES/messages.po to app/translations/sv/LC_MESSAGES/messages.mo
compiling catalog app/translations/ru/LC_MESSAGES/messages.po to app/translations/ru/LC_MESSAGES/messages.mo
compiling catalog app/translations/pt/LC_MESSAGES/messages.po to app/translations/pt/LC_MESSAGES/messages.mo
compiling catalog app/translations/fa/LC_MESSAGES/messages.po to app/translations/fa/LC_MESSAGES/messages.mo
compiling catalog app/translations/nl/LC_MESSAGES/messages.po to app/translations/nl/LC_MESSAGES/messages.mo
compiling catalog app/translations/hu/LC_MESSAGES/messages.po to app/translations/hu/LC_MESSAGES/messages.mo
compiling catalog app/translations/lt/LC_MESSAGES/messages.po to app/translations/lt/LC_MESSAGES/messages.mo
compiling catalog app/translations/nb_NO/LC_MESSAGES/messages.po to app/translations/nb_NO/LC_MESSAGES/messages.mo
compiling catalog app/translations/zh_Hans/LC_MESSAGES/messages.po to app/translations/zh_Hans/LC_MESSAGES/messages.mo
compiling catalog app/translations/gsw/LC_MESSAGES/messages.po to app/translations/gsw/LC_MESSAGES/messages.mo
compiling catalog app/translations/ro/LC_MESSAGES/messages.po to app/translations/ro/LC_MESSAGES/messages.mo
compiling catalog app/translations/ca/LC_MESSAGES/messages.po to app/translations/ca/LC_MESSAGES/messages.mo
compiling catalog app/translations/zh_Hant/LC_MESSAGES/messages.po to app/translations/zh_Hant/LC_MESSAGES/messages.mo
compiling catalog app/translations/is/LC_MESSAGES/messages.po to app/translations/is/LC_MESSAGES/messages.mo
compiling catalog app/translations/de/LC_MESSAGES/messages.po to app/translations/de/LC_MESSAGES/messages.mo

added 3 packages, changed 1 package, and audited 83 packages in 2s

16 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> wizarr@1.0.0 build:css
> npx @tailwindcss/cli -i src/style.css -o css/main.css

≈ tailwindcss v4.1.10

Done in 389ms
Creating app
Wizarr version: 🧙‍♂️ dev
Initialising extensions
Finished Initialising app
2025-06-27 09:38:53,753  INFO      Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 20250618b_drop_old_library_unique -> 20250619_add_wizard_step_table, add wizard_step table
  ✔️   Updated Wizarr to v2025.6.5
  ✔️   Started Wizarr
  ⏳   Cleaning Uprm: cannot remove '/tmp/2025.6.5.zip': No such file or directory

[ERROR] in line 63: exit code 0: while executing command rm /tmp/"$RELEASE".zip
```

checking tmp path: No .zip file is present in there


### 🖼️ Additional context (optional).

_No response_

--- a/ct/wizarr.sh
+++ b/ct/wizarr.sh
@@ -60,7 +60,6 @@ function update_script() {
 
     msg_info "Cleaning Up"
     rm -rf "$BACKUP_FILE"
-    rm /tmp/"$RELEASE".zip
     msg_ok "Cleanup Completed"
     msg_ok "Update Successful"
   else

